### ISSUE TITLE
Add LINZ aerial photography tiles in Online map source.

### ISSUE THREAD

**Issue by LexaNz** [osmandapp/OsmAnd#10506]
# 🚀 feature request

### Description

Add LINZ (Land Information New Zealand) aerial photo tiles in Online map in OsmAnd 

### Describe the solution you'd like

All info about usage is here : https://www.linz.govt.nz/data/linz-data/linz-basemaps/how-use-linz-basemaps-apis#xyz

**Comment by xmd5a2** [comment]
According to https://www.linz.govt.nz/data/linz-data/linz-basemaps/how-use-linz-basemaps-apis#xyz api key is required
`https://basemaps.linz.govt.nz/v1/tiles/aerial/3857/9/502/321.png?api={api_key}`
We can't add this source to our default sources because of that. But you can obtain api key and add this tile source manually via OsmAnd.

**Comment by LexaNz** [comment]
Yeah an API KEY is required, but this it's free and possible to get one for the application itself rather than to let the user add it on it's own.

Have a look on 'developer api key'
https://www.linz.govt.nz/data/linz-data/linz-basemaps/get-started-linz-basemaps

Other applications already use this source of tiles be default : 
https://play.google.com/store/apps/details?id=com.atlogis.nzmaps.free&hl=en
